include/beast/websocket/impl/read.ipp

```diff
@@ -511,23 +513,28 @@ operator()(error_code ec,
                     ec = boost::asio::error::operation_aborted;
                     goto upcall;
                 }
-                d.state = do_pong;
-                break; // VFALCO fall through?
 
```

Comment:
I would add a properly aligned comment here:
````
             // [[fallthrough]];
````
The `[[fallthrough]]` attribute is coming in C++17.